lib/ast.js

```diff
@@ -629,9 +629,14 @@ var AST_Switch = DEFNODE("Switch", "expression", {
     }
 }, AST_Block);
 
-var AST_SwitchBranch = DEFNODE("SwitchBranch", null, {
+var AST_SwitchBranch = DEFNODE("SwitchBranch", "body", {
     $documentation: "Base class for `switch` branches",
-}, AST_Block);
+    _walk: function(visitor) {
```

Comment:
Already changed, but that was code from yesterday before other solutions came in.